packages/utils/src/index.js

```diff
@@ -127,3 +129,24 @@ export function hasEntity(
   const entity = this.getCurrentEntity(editorState)
   return entity && entity.getType() === entityType
 }
+
+export function getCharCount(editorState: EditorState): number {
```

Comment:
does this pass flow?

Reply:
Currently, `@djsp/utils` can not pass flow. Looks like missing some files in flow-lib